engine/table/src/main/java/io/deephaven/db/tables/utils/kosak/parsers/ZamboniParser.java

```diff
@@ -0,0 +1,59 @@
+package io.deephaven.db.tables.utils.kosak.parsers;
+
+import io.deephaven.db.tables.utils.kosak.context.ParseContext;
+import io.deephaven.db.tables.utils.kosak.sinks.Sink;
+
+import java.util.function.Supplier;
+
+public abstract class ZamboniParser<TARRAY> {
```

Comment:
Another experiment comment: this file is going to be renamed also, not only moved.  We'll see if github preserves the comment.

Reply:
Comment still here anyway.